contacts.html

```diff
@@ -1 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
 
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./assets/css/contact.css">
+    <script src="./assets/js/firebase-connect.js"></script>
+    <script src="./assets/js/create-contact.js"></script>
+    <title>Contacts</title>
+</head>
+
+<body>
+
+    <main>
+        <div class="add-contact-wrapper">
+            <div class="add-contact-container">
+                <div class="left-side">
+                    <img src="./assets/img/logo-white.svg" alt="logo">
+                    <h2>Add contact</h2>
+                    <span>Task are better with a team!</span>
+                    <span class="border-line"></span>
+
+                </div>
+                <div class="right-side">
+                    <div class="person-img-container">
+                        <div class="person-img">
+                            <img src="./assets/img/person.svg" alt="person">
+                        </div>
+                    </div>
+                    <div class="form-elements">
+                        <div class="close-button-container">
+                            <img src="./assets/img/close.svg" alt="close">
+                        </div>
+
+                        <label for="name" class="form-label">
+                            <input type="text" id="name" name="name" placeholder="Name">
+                            <img src="./assets/img/person-gray.svg" alt="person">
+                        </label>
+
+                        <label for="email" class="form-label">
+                            <input type="text" id="email" name="email" placeholder="Email">
+                            <img src="./assets/img/mail.svg" alt="person">
+                        </label>
+
+                        <label for="phone" class="form-label">
+                            <input type="text" id="phone" name="phone" placeholder="Phone">
+                            <img src="./assets/img/call.svg" alt="person">
+                        </label>
+                        
+                        <div class="buttons">
+                            <button class="cancel-button">Cancel</button>
+                            <button class="create-contact-button" onclick="createContact()">Create contact</button>
+                        </div>
```

Comment:
hover und click-state von den buttons sieht noch nicht ganz richtig aus (der Schatten z.B.), bitte noch anpassen und prüfen, dass es wie in Figma aussieht: 
![image](https://github.com/user-attachments/assets/595ebf63-07e6-4878-a020-eedaa3886653)

Reply:
ich habe gemacht :)